### DISCUSSION TITLE
Loading fonts failed

### DISCUSSION THREAD

**Discussion by divdax** [spatie/laravel-google-fonts#30] [Bugs]
In my .env i use `https://project.test`. When loading fonts it tries to load the fonts from `http://project.test` (no https). Files exists in my storage and its linked properly.

**Comment by domnantas** [comment]
I am getting the same issue. `APP_URL` is using `https://`, however the local `url` for fonts is generated with `http://`

**Comment by drbourbon** [comment]
Same issue here
